code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -183,6 +183,82 @@
 
 	return TRUE
 
+/obj/item/borg/upgrade/abductorengi
```

Comment:
minor nitpick. Can you make this snake_case? Bit easier to read like that. Same for the other path